### PR TITLE
Adding product list from fetch

### PR DESCRIPTION
**Pull request by ChrisBerg03**
This need to be review and fixed abit. The styling need a bit of work and the filter does not yet work. need to be able to click the products to a single product view

### CODE REVIEW

File: index.html
Comment (JNettli) [inline index.html:1]: Thank you!! <3

File: allergy.html [hunk 1]
```diff
@@ -33,26 +33,20 @@ <h2> Nutritional Content</h2>
...
-    </nav>
+        <nav>
+            <a class="products" href="/product-list.html">
+                <img src="/assets/svg/products.svg" alt="icon of a box" />
+            </a>
+            <a class="cam">
+                <img src="/assets/svg/camera.svg" alt="icon of a camera" />
+            </a>
+            <a class="allergy" href="/profile.html">
+                <img
```
Comment (JNettli) [inline allergy.html:44]: Maybe make this img tag the same as the ones above?

**Review by JNettli — COMMENTED** [review]
Just one small thing to do here, but nothing that is disfunctional :)

**Review by ChrisBerg03 — COMMENTED** [review]
this is correct

**Comment by JNettli** [comment]
<3

**Pull request merged**